packages/allure-playwright/src/runtime.ts

```diff
@@ -51,6 +51,10 @@ export class AllurePlaywrightTestRuntime extends MessageTestRuntime {
     await test.info().attach(name, { path, contentType: options.contentType });
   }
 
+  async attachTrace(name: string, content: Buffer) {
```

Comment:
This function already does what we need, so there isn't necessary to add something to the commons' facade

Reply:
Also, add a test for the function